### ISSUE TITLE
Accessibility | Breadcrumb - aria-current=page is not implemented

### ISSUE THREAD

**Issue by fgeierst** [primefaces/primeng#19523]
### Describe the bug

> If the last link represents the current route, aria-current is added with "page" as the value.

While this [is documented like this](https://primeng.org/breadcrumb#router), `p-breadcrumb` does not expose the current page with `aria-current="page"`.

As a workaround we tried to add `aria-current` through passthrough on `itemLink`, but this is not working. In the rendered output, the attribute is not preserved, probably because they are overwritten by `routerLink` / `routerLinkActive`.

Note that aria-current is a requirement of the [APG Breadcrumb-Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/breadcrumb/#wai-ariaroles,states,andproperties).

### Reason for not contributing a PR

- [x] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/github-ynsptzwi?file=src%2Fapp%2Fapp.config.ts

### Environment

/

### Angular version

21.2.4

### PrimeNG version

v21

### Node version

24.13.0

### Steps to reproduce the behavior

1. Render a Breadcrumb with `routerLink` items.
2. Navigate to the route represented by the last breadcrumb item.
3. Inspect the rendered breadcrumb links.
4. Observe in the accessibility tab of dev tools that the current page link does not have `aria-current="page"`.

<img width="2541" height="1724" alt="Image" src="https://github.com/user-attachments/assets/c8f2e0c3-dab9-40dd-baf7-7474b54d9463" />

### Expected behavior

The current breadcrumb item should expose `aria-current="page"` automatically.
